spec/features/apo_spec.rb

```diff
@@ -81,7 +81,8 @@
     sign_in user, groups: ['sdr:administrator-role']
   end
 
-  it 'creates and edits an apo' do
+  # TODO: Figure out why this is flappy in CI and un-xit
+  xit 'creates and edits an apo' do
```

Comment:
can this be conditional based on the CI env variable?

Reply:
done